modules/con_notifications-configuration.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * argocd_instance/argo-cd-cr-component-properties.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="notifications-configuration_{context}"]
+= NotificationsConfiguration custom resource properties
+
+The `NotificationsConfiguration` resource is a Kubernetes custom resource (CR) that manages notifications in a Kubernetes cluster. In {gitops-title}, the `NotificationsConfiguration` CR allows users to add templates, triggers, services, and subscriptions resources to an Argo CD `Notifications` config map.
+
+When you create a cluster in {gitops-title} with notifications enabled, a `NotificationsConfiguration` CR is created by default with the name `default-notifications-configuration`.
+
+Any change made in the existing configuration of the `NotificationsConfiguration` CR is replicated in the Argo CD `Notifications` config map. For example, if the user adds  trigger configuration in the `NotificationsConfiguration` resource, this configuration is read, processed, and updated in the Argo CD `Notifications` config map.
+
+[IMPORTANT]
+====
+Any configuration changes must be updated in  the `default-notifications-configuration` CR. Custom resources created by the users for `NotificationsConfiguration` resource are not supported.
+
+Any modification to the Argo CD `argocd-notifications-cm` config map are overridden by the changes made in the `NotificationsConfiguration` CR.
+====
+
+The `NotificationsConfiguration` custom resource consists of the following properties.
+
+* `Templates` - Templates are used to generate the notification template message.
+* `Triggers` - Triggers are used to define the condition when a notification is sent to the user and the list of templates required to generate the message.
+* `Services` - Services are used to deliver message.
+* `Subscriptions` - Subscriptions contain centrally managed global application subscriptions.
+
+The default value for all these resources is `empty`.
+
+The following examples define how to add templates, triggers, services, and subscription resources to the Argo CD `argocd-notification-cm`` config map by using the `default-notifications-configuration` custom resource.
+
+.Example for `Templates`
+[source,yaml]
+----
+yaml
+apiVersion: argoproj.io/v1alpha1
+kind: NotificationsConfiguration
+metadata:
+ name: default-notifications-configuration <1>
+spec:
+ templates:
+  template.my-custom-template: | <2>
+    message: |
+     Application details: {{.context.argocdUrl}}/applications/{{.app.metadata.name}}.
+----
+<1> Default name of the `NotificationsConfiguration` CR in a cluster
+<2> Sample configuration for adding a custom template configuration to the the `NotificationsConfiguration` CR
```

Comment:
```suggestion
<1> Default name of the `NotificationsConfiguration` CR in a cluster.
<2> An example custom template configuration for the `NotificationsConfiguration` CR.
```
Adding missing periods. Plus I tried to add a little clarity to the second point so that it is more concise.

Reply:
Fixed.